src/redux/books/booksSlice.js

```diff
@@ -52,15 +47,44 @@ const booksSlice = createSlice({
         category: 'Action',
       };
 
-      state.books = [...state.books, newBook];
+      state.booklist = [...state.booklist, newBook];
     },
     removeBook: (state, action) => {
       const bookId = action.payload;
-      state.books = state.books.filter((book) => book.item_id !== bookId);
+      state.booklist = state.booklist.filter((book) => book.item_id !== bookId);
     },
   },
```

Comment:
- [ ] I noticed that you created two `removeBook` action creators instead of one. Based on the requirements, you ought to refactor your `addBook` and `removeBook` action creators with `createAsyncThunk` to include HTTP requests to the Bookstore API. Kindly refactor them instead of having two of each. 👍 🖥️

Reply:
Fixed.